WordPress/Classes/Networking/PlansRemote.swift

```diff
@@ -208,8 +209,14 @@ private func mapPlanFeaturesResponse(response: AnyObject) throws -> PlanFeatures
                     let planSpecificDescription = planInfo["description"] {
                         description = planSpecificDescription
                 }
+                    
+                // The slug currently returned for 'no ads' includes the text '/no-adverts.php'
+                // Remove this to 'normalize' all of the slugs
+                if let index = slug.characters.indexOf("/") {
```

Comment:
If we're not hardcoding the icons, is there another reason why we can't just store the slug as-is?